sdk/batch/Azure.Compute.Batch/tsp-location.yaml

```diff
@@ -1,3 +1,4 @@
 directory: specification/batch/Azure.Batch
-commit: dfa17e7ea91cf080769d7df32c349a35ccaa49ad
+commit: 06308c14162a70091d5ea4b079ae8740383a9e00
```

Comment:
this commit id is old dfa17e7ea91cf080769d7df32c349a35ccaa49ad is newer

Reply:
Let's bump to the latest - 4cb330be555a994ed507acf739e3349b7cb76d85